components/input_environmental_personal.py

```diff
@@ -27,11 +29,15 @@ def input_environmental_personal(selected_model):
         )
         inputs.append(input_filed)
 
+    if selected_model == 'Adaptive - ASHRAE 55':
```

Comment:
This is bad. We should not check a string but we should be referring to the class that defines the model

Reply:
The string should be replaced by Models.Adaptive.name